### DISCUSSION TITLE
Android - changing statusBar/navigationBar color

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#2478] [Q&A]
Is there a way to change statusBar or navitagionBar color through app descriptor xml or, even better, dynamically?

**Comment by marchbold** [comment]
You can use some of the `Display` functionality in our Application extension for this:

https://docs.airnativeextensions.com/asdocs/application/com/distriqt/extension/application/display/Display.html#setStatusBarColour()

**Reply by MalacTheLittle** [reply]
Thanks Michael, but I would rather avoid ANE solution if possible.
I found that it can be defined through [android manifest](https://stackoverflow.com/questions/37197235/change-status-bar-color-in-android), but not sure how to add it (if possible) to app descriptor xml.
